docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -169,218 +168,267 @@ Example
-provided, which in turn will produce one column per block in *X0*, so the final dimensions of matrix *X1* will be
-*Nx(LxK)*, where *L* is the number of header blocks in *X0* and *K* is the number of alpha values provided to the
-``RidgeReducer``. In practice, we can estimate a span of alpha values in a reasonable order of magnitude based on
-guesses at the heritability of the phenotype we are fitting.
+1. Model fitting, performed by the ``RidgeReducer.fit`` function, which fits multiple ridge models within each block :math:`x`.
+2. Model transformation, performed by the  ``RidgeReducer.transform`` function, which produces a new block matrix where each column represents the prediction of one ridge model applied within one block.
 
-Initialization
-==============
+This approach to model building is generally referred to as **stacking**. We call the starting block genotype matrix the **level 0** matrix in the stack, denoted by :math:`X^0`, and the output of the ridge reduction step the **level 1** matrix, denoted by :math:`X^1`. The ``RidgeReducer`` class is initialized with a list of ridge regularization values (here referred to as alpha). Since ridge models are indexed by these alpha values, the ``RidgeReducer`` will generate one ridge model per value of alpha provided, which in turn will produce one column per block in :math:`X^0`. Therefore, the final dimensions of :math:`X^1` for a single phenotype will be :math:`N \times (L \times K)`, where :math:`L` is the number of header blocks in :math:`X^0` and :math:`K` is the number of alpha values provided to the ``RidgeReducer``. In practice, we can estimate a span of alpha values in a reasonable order of magnitude based on guesses at the heritability of the phenotype we are fitting.
 
-When the ``RidgeReducer`` is initialized, it will assign names to the provided alphas and store them in a dictionary
-accessible as ``RidgeReducer.alphas``.
-
-Example
--------
+1. Initialization
+=================
 
-If alpha values are not provided, they will be generated during ``RidgeReducer.fit`` based on the unique number of
-headers *h* in the blocked genotype matrix *X0*, and a set of heritability values. These are only sensible if the
-phenotypes are on the scale of one.
+When the ``RidgeReducer`` is initialized, it assigns names to the provided alphas and stores them in a python dictionary accessible as ``RidgeReducer.alphas``. If alpha values are not provided, they will be generated during ``RidgeReducer.fit`` based on the number of unique headers in the blocked genotype matrix :math:`X^0`, denoted by :math:`h^0`, and a set of heritability values. More specifically,
 
 .. math::
 
-    \vec{\alpha} = h / [0.01, 0.25, 0.50, 0.75, 0.99]
+    \alpha = h^0\big[\frac{1}{0.99}, \frac{1}{0.75}, \frac{1}{0.50}, \frac{1}{0.25}, \frac{1}{0.01}\big]
+
+.. TODO: Clarify the following sentence:
+
+These are only sensible if the phenotypes are on the scale of one.
```

Comment:
These whats? Add a noun

Reply:
This was not my sentence. I beleive it is generally vague. Specifically the meaning of "the phenotypes are on the scale of one" is not clear. I had added the todo note to make it a better sentence after consulting with @LelandBarnard. 

But changed these => these values.